releasenotes/notes/36277.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: telemetry
+issue:
+  - 35750
+  - 36166
+  - 36521
+
+releaseNotes:
+  - |
+    **Fixed** an issue when setting distribution tracing to telemetry api, send a malfomed host header.
```

Comment:
```suggestion
    **Fixed** an issue where enabling tracing with telemetry API will cause malformed host header being used at the trace report request.
```

Reply:
I used a translator because I was not very good at English, but it looks better.
thank you